mkdocs.yml

```diff
@@ -0,0 +1,29 @@
+site_name: OPTIMADE-Python-Tools Documentation
+site_description: Documentation for the OPTIMADE-Python-Tools
+copyright: Built by OPTIMADE Development Team
+
+theme:
+  name: 'material'
+
+nav:
+  - Home: index.md
+  - Changelog: CHANGELOG.md
+  - Reference:
+      Models: reference/models.md
+
+site_url: https://materials-consortia.github.io/optimade-python-tools/
```

Comment:
Is this the right URL? `materials-consortia.github.io` redirects to `optimade.org`, so I guess this should either go to `optimade.org/optimade-python-tools` (if we get someone to update the domain settings for us) or to `optimade-python-tools.github.io`?

Reply:
Oh, good point. I don't know what happens to project repo pages when the org page is redirected...

Reply:
Any way we can test it before merging? Guess we need to enable Pages for this repo for one (will leave that to you when you're ready)

Reply:
`http://materials-consortia.github.io/optimade-python-tools` goes to a 404 under github pages so I think should work. I can't test the deploy until this is merged in as `mkdocs gh-deploy` uses the current `remote` to find the `gh-pages` branch. That being said, the docs will still work. Just the link back to the docs won't initially.